config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -285,17 +284,69 @@ config MBEDTLS_ENABLE_HEAP
 
 config MBEDTLS_HEAP_SIZE
     default 8192
+    
+# Enable PSA Crypto dependencies for Matter
 
-config NRF_SECURITY_ADVANCED
+if CHIP_CRYPTO_PSA
+
+config PSA_CRYPTO_DRIVER_CC3XX
+    default n
+
+config PSA_WANT_ALG_ECDSA
     default y
 
-config MBEDTLS_AES_C
+config PSA_WANT_ALG_ECDH
     default y
 
-config MBEDTLS_ECP_C
+config PSA_WANT_KEY_TYPE_ECC_KEY_PAIR
     default y
 
-config MBEDTLS_ECP_DP_SECP256R1_ENABLED
+config MBEDTLS_PSA_CRYPTO_STORAGE_C
+    default y if !BUILD_WITH_TFM
+
+config PSA_WANT_GENERATE_RANDOM
```

Comment:
I'm going to remove most of those kconfigs since they are already selected by the `OPENTHREAD_NRF_SECURITY_PSA` kconfig. The others are strictly related to nrfconnect.